products/compute/terraform.yaml

```diff
@@ -669,6 +669,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       optional_properties: |
         * `delete_default_routes_on_create`: If set to `true`, default routes (`0.0.0.0/0`) will be deleted
           immediately after network creation. Defaults to `false`.
+  NodeTemplate: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "node_template_basic"
+        primary_resource_id: "template"
+        vars:
+          template_name: "soletenant-tmpl"
+    properties:
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: false
+        default_from_api: true
+        custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+
```

Comment:
```suggestion
        description: |
          Region where nodes using this the node template will be created.
          If it is not provided, the provider region is used.
```

Reply:
(no idea if spacing will even almost be right)

Reply:
I'm going to suggest on your suggestion that s/this the/this/

Reply:
+ https://github.com/GoogleCloudPlatform/magic-modules/pull/1657/files#diff-fb4f76e7d870258668a3beac48bf164cR3190 in that case, that line was copied from there